packages/wdio-jasmine-framework/src/reporter.js

```diff
@@ -19,6 +19,17 @@ export default class JasmineReporter {
         suite.start = new Date()
 
         this.emit('suite:start', suite)
+
+        const hookBeforeAll = Object.assign({}, suite)
+        hookBeforeAll.type = 'hook'
+        hookBeforeAll.description = '"before all" hook'
+        this.emit('hook:start', hookBeforeAll)
```

Comment:
same as with hook:end, not sure that we always have before/after All and not sure if it's only one.

Reply:
See comment from before here as well ;-)